run_data_exploration.py

```diff
@@ -163,22 +146,69 @@ def get_enfsi_lrs_cacheable():
                     all_lrs_per_year[group] += zip(X1, X2)
                 else:
                     group = rater
-                cllrs.append([rater, group, round(calculate_cllr(list(
-                    X1), list(X2)).cllr, 4)])
+                cllr_results = calculate_cllr(list(
+                    X1), list(X2))
+                cllrs.append([rater, group, round(cllr_results.cllr, 4),
+                              round(cllr_results.cllr_min, 4)])
     for group, values in all_lrs_per_year.items():
         lrs1, lrs2 = zip(*values)
-        cllrs.append([group, group + '-all', round(calculate_cllr(list(
-            lrs1), list(lrs2)).cllr, 4)])
+        cllr_results = calculate_cllr(list(lrs1), list(lrs2))
+        cllrs.append([group, group + '-all', round(cllr_results.cllr, 4),
+                      round(cllr_results.cllr_min, 4)])
+    return pd.DataFrame(cllrs, columns=['rater', 'group', 'cllr', 'cllr_min'])
 
-    df_cllr = pd.DataFrame(cllrs,
-                           columns=['rater', 'group', 'cllr'])
-    st.altair_chart(alt.Chart(df_cllr, width=400).mark_circle(
-        size=20).encode(x='group', y='cllr'))
+if len(latest_lr_csv) == 0 or latest_exp_csv[:19] != latest_lr_csv[:19]:
+    st.markdown('No LR results available.')
```

Comment:
Ik zie dat als je 'enfsi' niet als dataset kiest, hij hier alsnog doorkomt. Er wordt dan wel een csv-tje gemaakt, maar met alleen kolomnamen.

Reply:
zal een nieuw taakje maken

Reply:
ok, ook toegevoegd dat we alleen een lr csv maken als we enfsi data hebben

Reply:
Top